### PR TITLE
build(deps): weekly dependency update; reduce dependabot frequency

### PR DESCRIPTION
**Pull request by MthwRobinson**
### Summary

Bumps dependencies and decreases the frequency of dependabot updates.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -3,11 +3,11 @@ updates:
   - package-ecosystem: "pip"
     directory: "/requirements"
     schedule:
-      interval: "weekly"
+      interval: "monthyl"
```
Comment (qued) [inline .github/dependabot.yml:6]:
```suggestion
      interval: "monthly"
```

File: unstructured/partition/email.py [hunk 2]
```diff
@@ -33,7 +33,6 @@
 
 
 def _parse_received_data(data: str) -> List[Element]:
-
```
Comment (qued) [inline unstructured/partition/email.py:36]: Did a linting standard change or something?

**Review by qued — APPROVED** [review]
LGTM, just a typo and a question.

File: unstructured/partition/email.py (hunk 2 above)
Comment (MthwRobinson) [reply unstructured/partition/email.py:36]: Yeah this was due to the `black` version bump.

**Pull request merged**